packages/image_picker/image_picker_for_web/lib/image_picker_for_web.dart

```diff
@@ -171,19 +179,22 @@ class ImagePickerPlugin extends ImagePickerPlatform {
   ///
   /// See https://caniuse.com/#feat=html-media-capture
   @visibleForTesting
-  Future<List<XFile>> getFiles({
-    String? accept,
-    String? capture,
-    bool multiple = false,
-  }) {
+  Future<List<XFile>> getFiles(
+      {String? accept,
+      String? capture,
+      bool multiple = false,
+      double? maxWidth,
+      double? maxHeight,
+      int? imageQuality}) {
```

Comment:
(Do not accept the resize image parameters any further down than the getImage/getImages method)